### ISSUE TITLE
Update the DTA sample resources in the repo for the new ini format

### ISSUE THREAD

**Issue by alexlambson** [CnCNet/xna-cncnet-client#354]
Running the repo using the sample resources is no longer possible due to the ini update. Once the ini update is complete, we'll need to update the sample resources with the updated DTA theme resources.

Low priority because I can just use the yuri theme that's been updated. This is mainly for new devs.

**Comment by SadPencil** [comment]
https://github.com/CnCNet/xna-cncnet-client/pull/588
